src/vendor/perfherder.js

```diff
@@ -52,11 +52,18 @@ const getFramework = async combo => {
           let lowerIsBetter = lower_is_better;
           let unit = 'Score';
 
-          if (lower_is_better === undefined) {
-            if (suite.endsWith('-power')) {
-              lowerIsBetter = true;
-              unit = 'mAh';
-            } else if (
+          if (suite.includes('youtube-playback')) {
+            lowerIsBetter = true;
+            unit = 'count';
+
+            if (exists(test) && test.includes('%')) {
+              unit = '';
+            }
+          } else if (suite.endsWith('-power')) {
+            lowerIsBetter = true;
+            unit = 'mAh';
+          } else if (lower_is_better === undefined) {
+            if (
```

Comment:
the `unit` does not come out of perfherder, so we add them here.  A lot more work is required: Like separating the desired label from the measurement units; which is best describe in a `Domain`